pwa/src/components/dashboardCard/DashboardCard.tsx

```diff
@@ -21,9 +21,18 @@ export interface DashboardCardProps {
   isEnabled?: boolean | undefined;
   lastRun?: string;
   lastCall?: string;
+  deleteOnClick?: (e?: any) => void;
```

Comment:
We should type this property. Instead of an optional `e?: any`, it should have the click event _and_ not be optional, like so:

```
deleteOnClick: (e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent<HTMLButtonElement>) => void;
```